components/textfield/index.css

```diff
@@ -13,23 +13,24 @@ governing permissions and limitations under the License.
 @import '../commons/index.css';
 
 :root {
-  --spectrum-textfield-padding-top: 3px;
-  --spectrum-textfield-padding-bottom: 5px;
+  --spectrum-textfield-quiet-invalid-background-position: 100% 50%;
 
-  /* Todo fix in DNA */
-  --spectrum-textfield-quiet-border-radius: 0;
+  --spectrum-textarea-height: auto;
```

Comment:
This is coded as `--spectrum-alias-single-line-height`, but it actually needs to be auto for our use case.